test/BUILD.gn

```diff
@@ -1126,6 +1126,7 @@ test("brave_browser_tests") {
       "//brave/app/theme:brave_theme_resources_grit",
       "//brave/app/theme:brave_unscaled_resources_grit",
       "//brave/browser/sharing_hub:browser_tests",
+      "//brave/browser/storage_access_api:browser_tests",
```

Comment:
we are not supposed to be adding new tests to this list. Why can't this be implemented as a `PlatformBrowserTest`?

Reply:
@AlexeyBarabash I thought we got rid of this and were only using the exceptions list now which has code owner?

Reply:
I guess tests that have functionality guarded by `!IS_ANDROID` would still go here, but I don't think that is the case for most of these

Reply:
Updated in 1ae6c846810 + fixup c51a389d8ec